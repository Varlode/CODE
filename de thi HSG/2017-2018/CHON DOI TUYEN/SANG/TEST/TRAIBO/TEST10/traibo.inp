2
8 2
100
300
500
700
900
1100
1300
1500
6 3
10000
50000
90000
130000
170000
210000