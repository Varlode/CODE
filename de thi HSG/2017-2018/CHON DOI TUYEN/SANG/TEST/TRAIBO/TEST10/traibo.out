1400
80000